Manager/src/main/java/io/raspberrywallet/manager/cryptography/crypto/algorithms/AlgorithmFactory.java

```diff
@@ -9,15 +9,15 @@
     
```

Comment:
`Factory` is highly misleading word, because it's not factory :D I suggest something like `AlghoritmInfo` or `AlghoritmParams`